Base/usr/share/man/man1/rev.md

```diff
@@ -0,0 +1,27 @@
+## Name
+
+rev - reverse lines
+
+## Synopsis
+
+```*sh
+$ rev [files...]
+```
+
+## Description
+
+`rev` reads the specified files line by line, and prints them to standard
+output in reverse. If no files are specifed, then `rev` will read from standard
+input.
+
+## Examples
+
```

Comment:
An example like the following would make it a bit more explicit?

```sh
$ cat foo
line1
line2
$ rev foo
1enil
2enil
```

Reply:
Good idea. I did this for both examples.